Upgrade docs for CanJS 6

m-mujica opened the pull request:
<img width="959" alt="Screen Shot 2019-09-30 at 1 00 14 PM" src="https://user-images.githubusercontent.com/724877/65895975-af6e3600-e382-11e9-978b-6be5b5175283.png">

phillipskevin commented on docs/can-stache-route-helpers.md:
```diff
@@ -47,22 +51,29 @@
+          <li>
+            <a {{# routeCurrent(page='recipe' id=5) }}class="active"{{/ routeCurrent }}
+              href="{{ routeUrl(page='recipe' id=5) }}">
+              {{ recipe.name }}
```
`this.recipe.name`

phillipskevin commented on docs/route-url.html:
```diff
@@ -1,31 +1,32 @@
 
-<script type='text/stache' id='demo-html'>
-<pre>url={{ routeUrl(undefined,true) }}</pre>
+	<a {{^ routeCurrent(page='home') }}href="{{ routeUrl(page='home') }}"{{/routeCurrent}}>
```
`{{/ routeCurrent }}`

phillipskevin commented on docs/route-url.html:
```diff
@@ -1,31 +1,32 @@
-	href="{{ routeUrl(page='home') }}"
-	{{/routeCurrent}}
->home</a>
+	<a {{^routeCurrent(page='restaurants')}}href="{{routeUrl(page='restaurants')}}"{{/routeCurrent}}>
```
`{{/ routeCurrent }}`

phillipskevin commented on docs/route-url.html:
```diff
@@ -1,31 +1,32 @@
-can.route(':page',{page: 'home'});
-can.route.start();
-$("body").append( can.view('demo-html',route) );
+	can.route(":page",{ page: "home" });
```
This is an old thing, but can you change this to `can.route("{page}", { page: "home" });`?

phillipskevin commented on docs/routeCurrent.md:
```diff
@@ -17,33 +17,36 @@
+      static view = `
+        <a {{# routeCurrent(page='recipe', true) }}class="active"{{/ routeCurrent }}
+          href="{{ routeUrl(page='recipe' id=5) }}">
+          {{ recipe.name }}
```
`this.recipe.name`

phillipskevin commented on docs/routeUrl.md:
````diff
@@ -14,28 +14,32 @@
 
   ```html
-  <a href='#!&page=recipe&id=5'>{{recipe.name}}</a>
+  <a href="#!&page=recipe&id=5">{{ recipe.name }}</a>
````
`this.recipe.name`

phillipskevin reviewed (changes requested):
A few small things.

m-mujica committed:
Upgrade docs for CanJS 6

m-mujica committed:
Fix example code

`recipe` property was returning an array, but the view is expecting
an object instead.
